### PR TITLE
Add JsMethod for calculateThumbnail

### PR DESCRIPTION
**Pull request by kevodwyer**

### CODE REVIEW

File: src/peergos/shared/user/fs/FileWrapper.java [hunk 1]
```diff
@@ -1000,6 +1000,27 @@ public static int getNumberOfChunks(long size) {
                 );
     }
 
+    @JsMethod
+    public CompletableFuture<Boolean> calculateThumbnail(NetworkAccess network, Crypto crypto) {
```
Comment (ianopolous) [inline src/peergos/shared/user/fs/FileWrapper.java:1004]: calculate suggests to me that something is returned. Maybe "updateThumbnail"?

File: src/peergos/shared/user/fs/FileWrapper.java [hunk 2]
```diff
@@ -1000,6 +1000,27 @@ public static int getNumberOfChunks(long size) {
...
+
+        BiFunction<Snapshot, Committer, CompletableFuture<Pair<Snapshot, Boolean>>> updateThumbnail =
```
Comment (ianopolous) [inline src/peergos/shared/user/fs/FileWrapper.java:1006]: Any reason not to inline this lambda?

Comment (kevodwyer) [reply src/peergos/shared/user/fs/FileWrapper.java]: now inlined

File: src/peergos/shared/user/fs/FileWrapper.java (hunk 1 above)
Comment (kevodwyer) [reply src/peergos/shared/user/fs/FileWrapper.java:1004]: updateThumbnail is already taken (used as part of updateProfile)

Comment (ianopolous) [reply src/peergos/shared/user/fs/FileWrapper.java:1004]: That's annoying, maybe calculateAndUpdateThumbnail? Probably existing updateThumbnail should have been called setThumbnail

Comment (kevodwyer) [reply src/peergos/shared/user/fs/FileWrapper.java]: done

**Review by ianopolous — APPROVED** [review]

**Pull request merged**